lib/hls/hls_parser.js

```diff
@@ -1925,10 +1925,11 @@ shaka.hls.HlsParser = class {
   /**
    * Get the spatial audio information for an HLS audio track.
    * In HLS the channels field indicates the number of audio channels that the
-   * stream has (eg: 2). In the case of Dolby Atmos, the complexity is
+   * stream has (eg: 2). In the case of Dolby Atmos (EAC-3), the complexity is
    * expressed with the number of channels followed by the word JOC
    * (eg: 16/JOC), so 16 would be the number of channels (eg: 7.3.6 layout),
-   * and JOC indicates that the stream has spatial audio.
+   * and JOC indicates that the stream has spatial audio. For Dolby AC-4 ATMOS,
+   * it's necessary search ATMOS word.
    * @see https://developer.apple.com/documentation/http_live_streaming/hls_authoring_specification_for_apple_devices/hls_authoring_specification_for_apple_devices_appendixes
```

Comment:
When I click that link it goes to the wrong section of the docs. Maybe they changed the URL structure of their docs, I see the correct section now has dashes rather than underscores.

https://developer.apple.com/documentation/http-live-streaming/hls-authoring-specification-for-apple-devices-appendixes

Also, is there any doc that mentions this ATMOS keyword in the CHANNELS string? I don't see that in these Apple docs, and it'd be nice to have a reference to go with this one that mentions the JOC keyword.

Reply:
That's correct, because Apple doesn't support AC-4 yet. The correct documentation is on Dolby's website, I've added a link to it.